jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-install.yaml

```diff
@@ -41,6 +41,9 @@
           name: 'dryRun'
           default: false
           description: '[OPTIONAL][Test] Dry run only, only prints what it would do!'
+      - string:
+          name: 'additionalParameters'
+          description: '[OPTIONAL] Comma separated list of additional parameters e.g amq_streams: true.'
```

Comment:
Would a text type parameter work better here? Could split on new line for each param.

Reply:
Thanks. It sounds good. I'll check

Reply:
Updated. It looks better and the params are separated by a new line. Thanks